packages/storybook-html/src/components/header/header_no-local-search.html

```diff
@@ -1,60 +1,41 @@
-            <div class="logo">
-                <a class="logo--large" href="https://www.uq.edu.au/"><img alt="The University of Queensland" src="https://static.uq.net.au/v11/logos/corporate/uq-lockup-landscape--reversed.svg"></a>
-                <a class="logo--small" href="https://www.uq.edu.au/"><img alt="The University of Queensland" src="https://static.uq.net.au/v11/logos/corporate/uq-logo--reversed.svg"></a>
-            </div>
-            <nav class="menu-global">
-                <ul>
-                    <li><a href="https://contacts.uq.edu.au/">Contacts</a></li>
-                    <li><a href="https://future-students.uq.edu.au/">Study</a></li>
-                    <li><a href="https://maps.uq.edu.au/">Maps</a></li>
-                    <li><a href="https://www.uq.edu.au/news/">News</a></li>
-                    <li><a href="https://www.uq.edu.au/events/">Events</a></li>
-                    <li><a href="https://jobs.uq.edu.au/">Jobs</a></li>
-                    <li><a href="https://www.library.uq.edu.au/">Library</a></li>
-                    <li><a href="https://giving.uq.edu.au/">Give now</a></li>
-                    <li><a href="https://my.uq.edu.au/">my.UQ</a></li>
-                    <li class="menu-global__search-toggle">
-                        <button class="search-toggle__button">Toggle Search</button>
-                    </li>
-                </ul>
-            </nav>
+        <div class="uq-header__logo">
+            <a class="uq-header__logo--large" href="https://www.uq.edu.au/"><img alt="The University of Queensland" src="https://static.uq.net.au/v11/logos/corporate/uq-lockup-landscape--reversed.svg"></a>
+            <a class="uq-header__logo--small" href="https://www.uq.edu.au/"><img alt="The University of Queensland" src="https://static.uq.net.au/v11/logos/corporate/uq-logo--reversed.svg"></a>
         </div>
-        <div class="nav-search">
-            <form class="nav-search__form" action="https://www.uq.edu.au/search/" method="get" accept-charset="UTF-8">
-                <fieldset class="nav-search__wrapper">
-                    <legend class="hidden">
-                        <span class="nav-search__title">What are you looking for?</span>
-                    </legend>
-                    <span class="nav-search__title" tabindex="0">What are you looking for?</span>
-                    <div class="nav-search__scope" tabindex="0">Search all UQ websites</div>
-                </fieldset>
-                <div class="nav-search__query">
-                    <span class="search-query__wrapper">
+        <div class="uq-header__search-toggle">
+            <button class="uq-header__search-toggle-button">Search</button>
+        </div>
+        <div class="uq-header__search">
+            <form>
+                <fieldset>
+                    <legend class="uq-header__search-title">What are you looking for?</legend>
+                    <div class="uq-header__search-range">
```

Comment:
did you exclude the input on purpose here?

Reply:
I don't think we need the extra text "search all uq websites"

Reply:
Do you mean the radio input? Shouldn't be necessary when there is no selection needed for this search.

Reply:
then lets remove the label as well

Reply:
I thought it was there to let users know what sites they are searching but noticed it is not on UQ home page. Happy to remove, any thoughts @Bcrunk ?

Reply:
pushed up a commit with label removed